package.json

```diff
@@ -27,84 +28,83 @@
   },
   "dependencies": {
     "@catena-x/portal-shared-components": "^2.1.42",
-    "@emotion/react": "^11.11.1",
+    "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.0",
     "@hookform/error-message": "^2.0.1",
-    "@mui/icons-material": "^5.11.16",
-    "@mui/material": "^5.13.4",
+    "@mui/icons-material": "^5.15.14",
+    "@mui/material": "^5.15.14",
     "@react-hook/cache": "^1.1.1",
-    "@reduxjs/toolkit": "^1.9.5",
-    "@types/papaparse": "^5.3.7",
-    "axios": "^1.6.4",
+    "@reduxjs/toolkit": "^2.2.1",
+    "axios": "^1.6.8",
     "buffer": "^6.0.3",
-    "dayjs": "^1.11.8",
-    "i18next": "^22.5.1",
-    "i18next-browser-languagedetector": "^7.0.2",
-    "keycloak-js": "^23.0.7",
+    "dayjs": "^1.11.10",
+    "i18next": "^23.10.1",
+    "i18next-browser-languagedetector": "^7.2.0",
+    "keycloak-js": "^24.0.1",
```

Comment:
Please change back to the 23.0.7 version to be aligned with CX IAM version for the 24.05 release